charts/internal/seed-controlplane/charts/csi-driver-controller/templates/deployment-csi-driver-controller.yaml

```
@@ -154,6 +157,7 @@ spec:
         - --kubeconfig=/var/lib/csi-resizer/kubeconfig
         - --leader-election=true
         - --leader-election-namespace=kube-system
+        - --timeout={{ .Values.timeout }}
```

For `quay.io/k8scsi/csi-resizer:v0.5.0`, the flag is still `--csiTimeout` (it is changed to `--timeout` since external-resizer@1.0.0).

```suggestion
        - --csiTimeout={{ .Values.timeout }}
```

```
$ docker run quay.io/k8scsi/csi-resizer:v0.5.0 --help

  -csiTimeout duration
    	Timeout for waiting for CSI driver socket. (default 15s)
```

---

This is weird, it must be a `--timeout` see the code: https://github.com/kubernetes-csi/external-resizer/blob/ae8826a2ac345af7a9bf4a6c5103441ec9940476/cmd/csi-resizer/main.go#L49

---

Thanks for a hint https://github.com/kubernetes-csi/external-resizer/commit/77f4ef877e47c891aa52e62309e90ceaa40c5983